test/worker/convert/test_pdf2tifs.py

```diff
@@ -0,0 +1,33 @@
+import os
+import shutil
+
+import unittest
+from pathlib import Path
+
+from worker.convert.converter import convert_pdf_to_tifs
+
+
+class Pdf2TifsTest(unittest.TestCase):
+    resource_dir = os.environ['RESOURCE_DIR']
+    working_dir = os.environ['WORKING_DIR']
+    pdf_src = f'{resource_dir}/objects/pdf/e2e-testing.pdf'
+    pdf_path = f'{working_dir}/e2e-testing.pdf'
+    tif_path = '{}/{}.tif'
+
+    @classmethod
+    def setUpClass(cls):
```

Comment:
Jeder test sollte atomar sein, d.h. den erwarteten Zustand erst unmittelbar bevor er läuft herstellen und unmittelbar hinter sich aufräumen. Insofern sollte immer `setUp` gegenüber `setUpClass` bevorzugt werden, wenn es nicht gute Gründe dafür gibt. Die sehe ich hier aber nicht.

@rhenck Ich kann mich dunkel entsinnen, dass du `setUpClass` eingeführt hast. Was waren nochmal die Gründe?

Reply:
Ich habe eigentlich nur tearDownClass gebraucht um die generierten Files aufzuräumen. Finde es macht auch Sinn, wenn man mehrere Tests in einer Klasse hat, die Ähnliches testen, dass die sich dann auch setUpund tearDown teilen.

Reply:
Die Alternative wäre, dass man Dinge, die vor jedem Test laufen sollen in `setUp` packt und Dinge, die nur einen Test betreffen im Test selber macht.

`setUpClass` ist insofern etwas gefährlich, als nicht ausgeschlossen ist, dass andere Tests den State, den `setUpClass` herstellt, verändern. Wenn man `setUp` nutzt, kann man dort genau den Zustand herstellen, den alle Tests der Klasse erwarten.